src/courtroom.cpp

```diff
@@ -3552,6 +3556,13 @@ void Courtroom::on_music_search_edited(QString p_text)
   }
 }
 
+void Courtroom::on_music_search_return_pressed()
+{
+  if (ui_music_search->text() == "") {
+    ui_music_list->collapseAll();
```

Comment:
This is kind of a cool alternative to right click -> expand all, me like.

Reply:
Typically expand all can also be done with the numpad asterisk key.